packages/sampler/stories/qa/Popup.js

```diff
@@ -30,6 +30,59 @@ const PopupFromSelfOnlyContainer = Toggleable(
 	)
 );
 
+class PopupResumeFocusAfterOpenState extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			isPopup: false,
+			isNotification: false
+		};
+	}
+
+	handlePopup = () => {
+		this.setState({
+			isPopup: true
+		});
+
+		setTimeout(() => {
+			this.setState({
+				isPopup: false
+			});
+		}, 200);
+	}
+
+	handleClose = () => {
+		this.setState({
+			isPopup: false
+		});
+	}
+
+	handleNotification = () => {
+		this.setState({
+			isNotification: false
+		});
+	}
+
+	render () {
+		return (
+			<div>
+				<Button onClick={this.handlePopup}>open</Button>
```

Comment:
Do we need to add a paragraph with some text describing what should happen here?

Reply:
It is a good idea. I was going to write a tc for it anyway.  How about:
"Click on "open" so the Popup appears and disappears quickly. Then 5-way Spot 'hello'.
? 
Please modify as needed.